lib/api/apiUtils/object/getReplicationInfo.js

```diff
@@ -89,7 +89,8 @@ function getReplicationInfo(objKey, bucketMD, isMD, objSize, operationType,
     //   actions triggered by internal services, by design)
 
```

Comment:
We might want to update the comment to reflect this change. Aside from that LGTM

Reply:
Good point! Updated.